configdialog.cpp

```diff
@@ -45,7 +45,9 @@ BOOL ConfigDialog::command(WPARAM wparam, LPARAM lparam) {
 			return TRUE;
 		}
 		case IDC_DEFAULTS_BUTTON: {
-			m_current_config = cfg_defaults;
+			for (const ConfigOption &opt : Opts::All) {
+				m_current_config[opt] = opt.default_;
```

Comment:
This demonstrates what enumerating through all variants will look like if we do this strategy for other enums, like `PaletteName`.